### PR TITLE
Add Card component

### PR DESCRIPTION
**Pull request by lukix**
cardcomponent

### CODE REVIEW

File: src/components/Card/Card.style.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components';
+import { colors } from 'config';
+
+const separatingBorder = `solid 2px ${colors.gray[2]}`;
+const smallPadding = '8px';
+const bigPadding = '32px';
+
+export const header = styled.header`
```
[the remark this answers is not in the document]
Comment (lukix) [reply src/components/Card/Card.style.js]: Done

File: src/components/Card/Card.style.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components';
+import { colors } from 'config';
+
+const separatingBorder = `solid 2px ${colors.gray[2]}`;
+const smallPadding = '8px';
```
[the remark this answers is not in the document]
Comment (lukix) [reply src/components/Card/Card.style.js]: Yep. Done.

File: src/components/Card/Card.jsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+import styles, { header } from './Card.style';
+
+const Card = styled.div(styles.card);
+Card.Header = header;
+Card.Body = styled.div(styles.body);
```
[the remark this answers is not in the document]
Comment (lukix) [reply src/components/Card/Card.jsx]: Done

File: src/components/Card/Card.style.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components';
+import { colors } from 'config';
+
+const SEPARATING_BORDER = `solid 2px ${colors.gray[2]}`;
+const SMALL_PADDING = '8px';
+const BIG_PADDING = '32px';
+
+export const Card = styled.article`
```
Comment (whereiswolf) [inline src/components/Card/Card.style.js]: maybe `div` 😛  ?

Comment (lukix) [reply src/components/Card/Card.style.js]: Done :'(

**Pull request merged**